### PR TITLE
editorconfig: Increase line length limit to 88

### PR DESCRIPTION
**Pull request by auscompgeek**
This globally increases the line length limit to 88, for consistency with Black.

## Checklist

- [ ] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [ ] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [ ] I have not broken the cheatsheet

### CODE REVIEW

File: packages/common/src/ide/types/Paths.ts [hunk 1]
```diff
@@ -4,10 +4,9 @@ type Primitive = string | number | symbol;
 
 type GenericObject = Record<Primitive, unknown>;
 
-type Join<
-  L extends Primitive | undefined,
-  R extends Primitive | undefined,
-> = L extends string | number
+type Join<L extends Primitive | undefined, R extends Primitive | undefined> = L extends
+  | string
+  | number
```
Comment (auscompgeek) [inline packages/common/src/ide/types/Paths.ts:9]: Ouch, this is definitely less readable.

File: packages/cursorless-engine/src/languages/markdown.ts [hunk 2]
```diff
@@ -60,15 +55,12 @@ type HeadingMarkerType = typeof HEADING_MARKER_TYPES[number];
...
-): NodeFinder {
+function makeMinimumHeadingLevelFinder(headingType: HeadingMarkerType): NodeFinder {
   const markerIndex = HEADING_MARKER_TYPES.indexOf(headingType);
   return (node: SyntaxNode) => {
     return node.type === "atx_heading" &&
-      HEADING_MARKER_TYPES.indexOf(
-        node.firstNamedChild?.type as HeadingMarkerType,
-      ) <= markerIndex
+      HEADING_MARKER_TYPES.indexOf(node.firstNamedChild?.type as HeadingMarkerType) <=
+        markerIndex
```
Comment (auscompgeek) [inline packages/cursorless-engine/src/languages/markdown.ts:63]: This seems to be less readable, at least whilst prettier still wraps lines after binary operators.

**Review by auscompgeek — COMMENTED** [review]
Damn, I wasn't expecting so many lines to be just over the 80 character limit :smile:

Some of this looks a lot more readable, some is pretty meh. Some is a bit less readable though.

File: packages/cheatsheet/src/lib/components/CheatsheetListComponent.tsx [hunk 3]
```diff
@@ -6,16 +6,12 @@ type Props = {
   section: CheatsheetSection;
 };
 
-export default function CheatsheetListComponent({
-  section,
-}: Props): JSX.Element {
+export default function CheatsheetListComponent({ section }: Props): JSX.Element {
```
Comment (pokey) [inline packages/cheatsheet/src/lib/components/CheatsheetListComponent.tsx:9]: I find it slightly more readable to have less complexity per line

**Review by pokey — COMMENTED** [review]
I'm neutral to _very slightly_ negative.  @AndreasArvidsson if you're strongly positive I'll let myself be dragged along for the ride here 😄

**Comment by AndreasArvidsson** [comment]
I'm probably slightly positive. I do like longer lines, but at the same time I'm not always in love with how prettier decides to break lines.

**Comment by pokey** [comment]
Hmm ok, well if you're slightly positive and I'm slightly negative, I think I'll close, as I'm the one who has to spend his whole day in this code base 😄

Let's reopen if we start getting more commits from people who aren't me (whether that be you, or anyone else who is on a screen bigger than mine)

If you were more than slightly negative, I'd be happy to push it through, but I'm going with a weighted average here 😊

**Pull request closed without merging**